### PR TITLE
Lesson 3 Part 2: Bash

### PR DESCRIPTION
**Pull request by dmytro-moruha**
Wrote a bash script from HW part 2

### CODE REVIEW

**Comment by a-vodka** [comment]
- We have to copy all cpp files to temp dir
- You should define variable for temp_dir

**Comment by dmytro-moruha** [comment]
Copy all source files
Define variables for all paths and name
@marynasergiyenko,@a-vodka please check my PR

File: 01-Git_Bash_Make/script.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Bash script
+
+dir=/tmp/guesanumber
+arch=guesanumber.tar.gz
+rel=release
```
Comment (a-vodka) [inline 01-Git_Bash_Make/script.sh]: Dir, arch and rel variable have to be upper-case

**Comment by dmytro-moruha** [comment]
Make all variables UpCase
@marynasergiyenko,@a-vodka please check my PR

**Pull request merged**